e2e/testcafe-devextreme/package.json

```diff
@@ -3,30 +3,35 @@
   "version": "25.2.0",
   "scripts": {
     "test": "node ./runner.js",
-    "lint": "eslint --ext .js,.ts .",
+    "lint": "eslint",
     "update-failed-etalons": "node update_failed_etalons.mjs"
   },
   "devDependencies": {
     "@babel/eslint-parser": "catalog:eslint8",
     "@babel/plugin-transform-runtime": "7.19.6",
     "@testcafe-community/axe": "3.5.0",
     "@types/jquery": "3.5.29",
-    "@typescript-eslint/eslint-plugin": "catalog:eslint8",
-    "@typescript-eslint/parser": "catalog:eslint8",
     "axe-core": "4.10.3",
     "devextreme": "workspace:*",
     "devextreme-screenshot-comparer": "2.0.17",
     "devextreme-testcafe-models": "workspace:*",
-    "eslint": "catalog:eslint8",
-    "eslint-config-airbnb-typescript": "catalog:eslint8",
-    "eslint-plugin-i18n": "catalog:eslint8",
-    "eslint-plugin-import": "catalog:eslint8",
-    "eslint-plugin-no-only-tests": "catalog:eslint8",
-    "eslint-plugin-qunit": "catalog:eslint8",
+    "eslint": "catalog:",
+    "@eslint-stylistic/metadata": "catalog:",
+    "@eslint/eslintrc": "catalog:",
+    "@stylistic/eslint-plugin": "catalog:",
+    "@typescript-eslint/eslint-plugin": "catalog:",
+    "@typescript-eslint/experimental-utils": "5.62.0",
+    "@typescript-eslint/parser": "catalog:",
+    "eslint-config-devextreme": "1.1.6",
+    "eslint-migration-utils": "workspace:*",
+    "eslint-plugin-i18n": "catalog:",
+    "eslint-plugin-import": "catalog:",
+    "eslint-plugin-no-only-tests": "catalog:",
+    "eslint-plugin-testcafe": "0.2.1",
```

Comment:
Did we decide to use eslint-plugin-testcafe rules instead of devextreme/testcafe? If so, why?